scripts/create_index_files_simcap.py

```diff
@@ -36,39 +36,31 @@ def main(args):
 
     # withhold pumas without weather
     with open(Path(os.environ.get('BUILDINGS_BENCH','')) / 'metadata_dev' / 'pumas_without_weather.pkl', 'rb') as f:
-        withheld_pumas += pickle.load(f)
+        without_weather_pumas = pickle.load(f)
     print(f'Withheld pumas: {withheld_pumas}')
+    print(f'Without weather puma: {without_weather_pumas}')
 
     # 2 weeks heldout for val
     train_tmy_timerange = (pd.Timestamp('2018-01-01'), pd.Timestamp('2018-12-31'))
-    train_amy2018_timerange = (pd.Timestamp('2018-01-01'), pd.Timestamp('2018-12-17'))
-    val_timerange = (pd.Timestamp('2018-12-17'), pd.Timestamp('2018-12-31'))
+    # train_amy2018_timerange = (pd.Timestamp('2018-01-01'), pd.Timestamp('2018-12-17'))
+    # val_timerange = (pd.Timestamp('2018-12-17'), pd.Timestamp('2018-12-31'))
     
 
     for building_type_and_year, by in enumerate(building_years):
         by_path = time_series_dir / by / 'timeseries_individual_buildings'
 
         # building ids contained in the simcap df
-        bd_ids = pd.read_csv(output_dir / "simcap" / f"{by[:-10]}_simcap_10000.csv")["bldg_id"].values
+        bd_ids = pd.read_csv(output_dir / "simcap" / f"{by[:-10]}_simcap_300000.csv")["bldg_id"].values
         bd_ids = np.array(bd_ids)
-        # randomly split train (80%) and test (20%) sets
-        n = len(bd_ids)
-        mask = np.zeros(n).astype(bool)
-        mask[np.random.choice(n, int(n * 0.8), replace=False)] = True
-
-        train_idx = set(bd_ids[mask])
-        test_idx = set(bd_ids[~mask])
-        
-        if 'amy2018' in by:
-            train_hours = int((train_amy2018_timerange[1] - train_amy2018_timerange[0]).total_seconds() / 3600)
-            print(f'AMY2018 train hours: {train_hours}')
 
-            val_hours = int((val_timerange[1] - val_timerange[0]).total_seconds() / 3600)
-            print(f'AMY2018 Val hours: {val_hours}')
+        # randomly split train (60%) and val (20%) test (20%) sets
```

Comment:
this is OK for now, for "i.i.d. generalization" experiments. We need to also create some "OOD generalization" test splits for CLP^2, where we identify attributes we want to use to test systematic generalization (withholding combinations). My suggestion is to pick ranges of values (continuous) or certain buckets (discrete) from two attributes and withhold the intersection. I can help with this.